r/tests/testthat/test-filesystem.R

```diff
@@ -83,7 +83,6 @@ test_that("SubTreeFilesystem", {
   expect_is(st_fs, "FileSystem")
```

Comment:
So you should simply /-normalize `td` above. Should be easy :-)

Reply:
Should be!

Reply:
At worse you simply string-replace `\` with `/` (but only when on Windows :-)).

Reply:
That's where I ended up: https://github.com/apache/arrow/pull/5445/files#diff-3bd936ee39b8abe149e5399681610c75R255

Reply:
Ah, then you shouldn't call it `fs_relative_path`...